profile/README.md

```diff
@@ -17,97 +17,109 @@
 
 ### 📝 Docs
 
-| Website      | Description | Repo |
-| ----------- | ----------- | --- |
-|[docs.near.org](https://docs.near.org) | NEAR Developer Documentation |[near/docs](https://github.com/near/docs)
-|[nomicon.io](https://nomicon.io)| NEAR Protocol Specification Documentation | [near/neps](https://github.com/near/neps)
-|[near-nodes.io](https://near-nodes.io)| NEAR Node Documentation _(Validator, RPC, Archival)_ |[near/node-docs](https://github.com/near/node-docs)
-
+| Website                                | Description                                          | Repo                                                |
+|----------------------------------------|------------------------------------------------------|-----------------------------------------------------|
+| [docs.near.org](https://docs.near.org) | NEAR Developer Documentation                         | [near/docs](https://github.com/near/docs)           |
+| [nomicon.io](https://nomicon.io)       | NEAR Protocol Specification Documentation            | [near/neps](https://github.com/near/neps)           |
+| [near-nodes.io](https://near-nodes.io) | NEAR Node Documentation _(Validator, RPC, Archival)_ | [near/node-docs](https://github.com/near/node-docs) |
+ 
 ---
 
 ### ⛓️ Protocol
 
-| Name      | Description | Repo | Latest Release |
-| ----------- | ----------- | --- | --- |
-| nearcore | Reference implementation of NEAR Protocol  |[near/nearcore](https://github.com/near/nearcore)|[![Latest Release](https://img.shields.io/github/v/release/near/nearcore?label=)](https://github.com/near/nearcore/releases)
-| NEPs | NEAR Protocol Specifications and Standards  |[near/neps](https://github.com/near/neps)| ➖
+| Name     | Description                                | Repo                                              | Latest Release                                                                                                               |
+|----------|--------------------------------------------|---------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
+| nearcore | Reference implementation of NEAR Protocol  | [near/nearcore](https://github.com/near/nearcore) | [![Latest Release](https://img.shields.io/github/v/release/near/nearcore?label=)](https://github.com/near/nearcore/releases) |
+| NEPs     | NEAR Protocol Specifications and Standards | [near/neps](https://github.com/near/neps)         |                                                                                                                              |
+
+---
+
+### 🔗 Chain Abstraction
```

Comment:
@thisisjoshford this is the relevant part, the rest is just text formatting from my vscode